source/WorkingPractices/approvals.rst

```diff
@@ -40,6 +38,9 @@ A list of current code owners for each project can be found in
     able to guide your change to fit in with the bigger picture of what is
     happening.
 
+Code Owners are good candidates for choosing a SciTech reviewer, and you should
+consider this early in the process so that you can involve them in design decisions.
```

Comment:
In fact I'm struggling a bit with this appearing to duplicate what was said in the previous paragraph. Could you check this section makes sense and 'flows' nicely when read through in it's entirety.

Reply:
Have removed the extra reference to talking to people early - it was perhaps one too many